testsuite/domain/src/test/resources/base-server-config.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<server xmlns="urn:jboss:domain:community:20.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="urn:jboss:domain:community:20.0 wildfly-config_community_20_0.xsd">
+<server xmlns="urn:jboss:domain:community:20.0">
```

Comment:
This file is still using "community" name space

Reply:
This conversation was marked as resolved but I am still also seeing the community namespace here.

Reply:
Ignore that the GitHub view was showing an outdated diff even though I just opened the PR.